pkg/cmd/server/api/v1/types.go

```diff
@@ -377,6 +377,9 @@ type AssetConfig struct {
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL"`
 
+	// CLIDownloadPublicURLs holds links to download client tools, where key is the platform name and value is the download URL (optional)
```

Comment:
We may want to reduce the amount of info in that page, but an entry in the config file for CLI download links is a specific requirement from online. Links for docs already use the filters mechanism used everywhere on the web console.